dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
+        <button type="button" class="nav">Home</button>
+        <button type="button" class="nav">Projects</button>
+        <button type="button" class="nav">About Me</button>
+        <button type="button" class="nav">Contact</button>
+      </div>
+      <span class="perks">
+        Custom Designs · Marketing · Branding · And More
+      </span>
+      <div class="whois">
```

Comment:
**Lines 22-31:**

We should probably wrap this whole section in a parent `<div>`. I recommend this because these two boxes are meant to be two columns in the same box, as per the mockup.

Reply:
Gotcha!